buildkite/bazelci.py

```diff
@@ -3895,13 +3912,20 @@ def download_bep(self, dest_dir: str) -> str:
 
         # We cannot use `buildkite agent download *` since it cannot handle backslashes in Windows artifact paths.
         # Consequently, we just escape all backslashes and download files directly from GCS.
-        url = "/".join([LOG_BUCKET, self.job_id, self.relative_bep_path.replace("\\", "%5C")])
+        url = "/".join(
+            [LOG_BUCKET, self.job_id, self.relative_bep_path.replace("\\", ESCAPED_BACKSLASH)]
+        )
         try:
             return download_file(url, job_dir, _TEST_BEP_FILE)
         except:
             # TODO: handle exception
             return None
 
+    @property
+    def is_windows(self) -> bool:
+        # Dirty hack
+        return "\\" in self.relative_bep_path
```

Comment:
If you want to check whether this is running on windows use `os == 'nt'` instead.

Reply:
We process all BEP files across all platforms in a single job running on Linux, so the current os won't help.

Reply:
Well, it is dirty, but it is what it is.